src/features/amenities/ManageAmenities.tsx

```diff
@@ -67,68 +71,93 @@ export default function ManageAmenities() {
 	};
 
 	return (
-		<ContainerCard>
-			<ContainerCardTitle>Manage Amenities</ContainerCardTitle>
+		<>
+			<ContainerCard>
+				<ContainerCardTitle>Manage Amenities</ContainerCardTitle>
 
-			<Table<AmenityType>
-				dataSource={amenities}
-				rowKey="_id"
-				loading={loading}
-			>
-				<Table.Column<AmenityType>
-					title="Name"
-					dataIndex="name"
-					sorter={sortStringByProperty<AmenityType>('name')}
-				/>
-				<Table.Column<AmenityType>
-					title="Description"
-					dataIndex="description"
-					sorter={sortStringByProperty<AmenityType>('description')}
-				/>
-				<Table.Column<AmenityType>
-					title="Fee"
-					dataIndex="fee"
-					sorter={sortNumberByProperty<AmenityType>('fee')}
-				/>
-				<Table.Column<AmenityType>
-					title="Icon"
-					dataIndex="icon"
-					sorter={false}
-				/>
-				<Table.Column<AmenityType>
-					title="Created At"
-					dataIndex="createdAt"
-					sorter={sortDateByProperty<AmenityType>('createdAt')}
-					render={(createdAt) =>
-						dayjs(createdAt).format('DD MMM h:mm:ss A')
-					}
-				/>
-				<Table.Column<AmenityType>
-					title="Actions"
-					render={(value, amenity) => (
-						<>
-							<Space>
-								<Button
-									type="primary"
-									shape="circle"
-									icon={<EditFilled />}
-								/>
+				<div className="text-right mb-4 -mt-6">
+					<Button
+						type="primary"
+						onClick={() => setCreateAmenitiesVisible(true)}
+					>
+						Create
+					</Button>
+				</div>
 
-								<Button
-									type="primary"
-									shape="circle"
-									icon={<DeleteFilled />}
-									danger
-									onClick={() =>
-										amenity._id &&
-										deleteAmenity(amenity._id)
-									}
-								/>
-							</Space>
-						</>
-					)}
+				<Table<AmenityType>
```

Comment:
@harisrahman , Can we use map here or make it more generic as we are repeating Table.column .